core/src/mindustry/world/blocks/defense/MendProjector.java

```diff
@@ -56,21 +56,13 @@ public void setStats(){
             stats.add(Stat.boostEffect, (phaseBoost + healPercent) / healPercent, StatUnit.timesSpeed);
         }
     }
-
+    
```

Comment:
?

Reply:
Dunno why that happened or when I did that.